lib/renderer/init.ts

```diff
@@ -34,7 +33,9 @@ webFrameInit()
 // Process command line arguments.
 const { hasSwitch, getSwitchValue } = process.atomBinding('command_line')
 
-const parseOption = function (name, defaultValue, converter = value => value) {
+const parseOption = function (
+  name: string, defaultValue: any, converter: (value: any) => any = value => value
```

Comment:
The `converter` here can be typed as `string` and `T`.

```suggestion
  name: string, defaultValue: T, converter: (value: string) => T = value => value
```

Then `parseOption` is strongly typed

Reply:
Upon closer investigation, that doesn't work – and it makes sense, this here doesn't work: `(value: string) => T = value => value`